Dockerfile.j2

```diff
@@ -87,8 +94,11 @@ ENV TARGET_CC={{ target }}-gcc \
     TARGET_SYSROOT=/usr/{{ target }}/{{ target }}/sysroot/ \
     TARGET_C_INCLUDE_PATH=/usr/{{ target }}/{{ target }}/sysroot/usr/include/
 
-ENV CARGO_BUILD_TARGET={{ target | replace('ibm', 'unknown') }}
-ENV CARGO_TARGET_{{ target | replace('ibm', 'unknown') | replace('-', '_') | upper }}_LINKER={{ target }}-gcc
+ENV CARGO_BUILD_TARGET={{ target | replace('ibm', 'unknown') | replace('riscv64', 'riscv64gc') }}
+ENV CARGO_TARGET_{{ target | replace('ibm', 'unknown') | replace('-', '_') | replace('riscv64', 'riscv64gc') | upper }}_LINKER={{ target }}-gcc
+{% if platform.startswith('musllinux') -%}
+ENV CARGO_TARGET_{{ target | replace('ibm', 'unknown') | replace('-', '_') | upper }}_RUSTFLAGS='-C target-feature=+crt-static'
```

Comment:
https://peps.python.org/pep-0656/#rationale

> The musllinux platform tags only apply to Python interpreters dynamically linked against the musl libc and executed on the runtime shared library, on a Linux operating system. Statically linked interpreters or mixed builds with other libc implementations (such as glibc) are out of scope and not supported by platform tags defined in this document. Such interpreters should not claim compatibility with musllinux platform tags.

I don't think we should static link musl libc?

Reply:
OK, let me fix that.